dotnet/src/Experimental/Process.Core/ProcessVisualizationExtensions.cs

```diff
@@ -0,0 +1,159 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.SemanticKernel.Process;
```

Comment:
wondering if this should be in a .Utilities namespace since it is not necessarily always used

Reply:
perhaps move this file to a folder/namespace Tools here.

This is not quite utilities since it is more of an add-on